code/datums/datum.dm

```diff
@@ -57,3 +57,9 @@
 /datum/proc/Process()
 	SHOULD_NOT_SLEEP(TRUE)
 	return PROCESS_KILL
+
+// This is more or less a helper to avoid needing to cast extension holders to atom.
+// See /atom/get_recursive_loc_of_type() for actual logic.
+/datum/proc/get_recursive_loc_of_type(var/loc_type)
```

Comment:
Can you add the old name(s) to the comment here so people searching for it have a greater chance to find it?